fetchai/ledger/__init__.py

```diff
@@ -1,4 +1,10 @@
 # Ledger version that this API is compatible with
-__version__ = '0.9.0a1'
+__version__ = '0.9.0-a1'
```

Comment:
Yes this is going to be interesting. I think that this syntax might not be pip compatible!

Reply:
This pep suggests that this shouldn't be an issue:
https://www.python.org/dev/peps/pep-0440/#pre-release-separators

Reply:
👍